subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testsuites/dependencies/TestSuitesKotlinDSLDependenciesIntegrationTest.groovy

```diff
@@ -1748,7 +1707,7 @@ class TestSuitesKotlinDSLDependenciesIntegrationTest extends AbstractIntegration
             suites {
                 $suiteDeclaration {
                     dependencies {
-                        implementation(libs.bundles.groovy)
+                        implementation.bundle(libs.bundles.groovy)
```

Comment:
Should Groovy DSL be doing the same thing?

Reply:
I think it is? Can you show where it isn't?